### PR TITLE
Add configuration option for audit log

### PR DESCRIPTION
**Pull request by legionus**

### CODE REVIEW

File: inventory/byo/hosts.origin.example [hunk 1]
```diff
@@ -324,6 +324,10 @@ openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true',
 #
 # Disable management of the OpenShift Registry
 #openshift_hosted_manage_registry=false
+#
+# Enable audit logger. An OpenShift registry will extra records with
```
Comment (mfojtik) [inline inventory/byo/hosts.origin.example]: An OpenShift integrated registry will add extra records

**Comment by mfojtik** [comment]
@sdodson PTAL

File: inventory/byo/hosts.origin.example (hunk 1 above)
Comment (legionus) [inline inventory/byo/hosts.origin.example]: @mfojtik fixed

**Comment by detiber** [comment]
aos-ci-test

**Comment by sdodson** [comment]
I think the registry config only gets included when certain storage types are used so we'd want to change that.

**Comment by michaelgugino** [comment]
Closing due to inactivity.

**Pull request closed without merging**